test/services/ConsignmentServiceSpec.scala

```diff
@@ -221,6 +224,67 @@ class ConsignmentServiceSpec extends AnyWordSpec with MockitoSugar with BeforeAn
     getConsignmentDetails should be(GetConsignment(0, None))
   }
 
+  "getConsignmentFileMetadata" should {
+    "return consignment with file metadata when consignment ID and selected file ids are passed" in {
```

Comment:
Lower case "id" for consistency.

Reply:
Right, updated it everywhere in the file, thanks.